src/qt/rpcconsole.cpp

```diff
@@ -610,6 +610,7 @@ bool RPCConsole::eventFilter(QObject* obj, QEvent *event)
     if(event->type() == QEvent::KeyPress) // Special key handling
     {
         QKeyEvent *keyevt = static_cast<QKeyEvent*>(event);
+        const auto type = keyevt->type();
         int key = keyevt->key();
         Qt::KeyboardModifiers mod = keyevt->modifiers();
```

Comment:
Since we are refactoring this part of the code, how about we initialize the `key` and `mod` variables as **const** since they are unchanged in their lifetime.

```suggestion
        const int key = keyevt->key();
        const auto mod = keyevt->modifiers();
```

Reply:
Ok, I'll implement this suggestion if/when other changes will be required.